### ISSUE TITLE
Missing Settings?

### ISSUE THREAD

**Issue by mak5imka** [bvn-architecture/RevitBatchProcessor#62]
Hi @DanRumery , 
I just discovered your awesome tool and trying to make it work for me. Unfortunately I'm getting an odd error:

16:56:23 : 
16:56:23 : Session ID: <2020-06-19T14:56:23.493Z>
16:56:23 : 
16:56:23 : Log File:
16:56:23 : 
16:56:23 : 	C:\Users\Maksim Koenig\AppData\Local\BatchRvt\BatchRvt_20200619_165623_493.log
16:56:23 : 
16:56:23 : ERROR: No settings file specified or settings file not found.
16:56:23 : 
16:56:23 : Operation aborted.
16:56:23 : 

As far as I can tell, everything is specified in the GUI. Could you help me out please? Let me know if you need any further information.
[BatchRv_2.txt](https://github.com/bvn-architecture/RevitBatchProcessor/files/4805310/BatchRv_2.txt)

**Comment by DanRumery** [comment]
Yeah that does look odd. Are you just setting it up in the GUI but running it from the command-line? or both setting up and running from the GUI?

**Comment by mak5imka** [comment]
Yes, I'm just using the GUI. Tried different settings, but everything leads to the same error.

**Comment by DanRumery** [comment]
Which version of RBP are you using?

**Comment by mak5imka** [comment]
I had the same issue with 1.5.3 and 1.6.0 (installed right now)

**Comment by DanRumery** [comment]
hmmm where is the settings file currently saved to / loaded from?

**Comment by mak5imka** [comment]
Its saved in the same place as the script. I first tried on my onedrive folder and thought this might cause the problems. So I switched to a local folder and its the same.

**Comment by mak5imka** [comment]
Ha, I just tested with command line and it works! :D Awesome. Seems to be a problem with the GUI

**Comment by mak5imka** [comment]
I'll try on a different machine in my office (next week) and let you know. Thank you for reaching out to me!

**Comment by mak5imka** [comment]
One last questions: how can I make the tool use one revit session? This option is grayed out in the GUI. Can I modify the settings file to achieve this?

**Comment by DanRumery** [comment]
If you're running Dynamo scripts unfortunately RBP always runs separate Revit sessions. I wasn't able to find a way to "reset" Dynamo in the Revit session so I had to disable the option for them. If you instead use a Python task script exclusively then the option becomes available.
